src/EnergyPlus/ChillerExhaustAbsorption.hh

```diff
@@ -146,7 +146,6 @@ namespace ChillerExhaustAbsorption {
         int ExhTempLTAbsLeavingHeatingTempIndex; // index for exhaust potential less than thermal energy needed during heating
         std::string TypeOf;                      // Generator type
         std::string ExhaustSourceName;           // Generator type Name
-        bool oneTimeFlag;
```

Comment:
Removed flags like these, since oneTimeInit_new will be called from the PlantManager.cc after this.

Reply:
👍